scripts/Run_uncrustify_for_sources.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# 27. 10. 2016
+#
+SRC="./src"
+#
+where=`pwd`
+#
+# build the lists
+cd ${SRC}
+list_of_H=`ls *.h`
+list_of_C=`ls *.cpp`
+cd ${where}
+#
+mkdir -p results
+#
+for file in ${list_of_H} ${list_of_C}
+do
+  uncrustify -q -c ./forUncrustifySources.cfg -f ${SRC}/${file} -o results/${file}
+  diff ${SRC}/${file} results/${file} > /dev/null 2>&1
```

Comment:
A `cmp -s ${SRC}/${file} results/${file}` should be better here if you just want to compare both files without getting any metrics.
Additionally a check for errors should be done as well: 

> 0 if inputs are the same, 1 if different, 2 if trouble